### ISSUE TITLE
Buttons: Update hover and focus colors to meet color contrast requirements

### ISSUE THREAD

**Issue by nataliafitzgerald** [cfpb/design-manual#603]
## Buttons: Update hover and focus colors to meet color contrast requirements
Some button states currently do not meet color contrast requirements for accessibility and should be updated:

These include: 
- Disabled button (Gray text on Gray 20 background)
- Primary button: Hover and focus state (White on Pacific 80)
- Destructive action button: Hover and focus state (White on Red 80)

Recommendations (some are still works in progress):
 
## Disabled button 
| Current (Gray #43484e) | Recommendation (Dark Gray #d2d3d5) | 
| --- | --- | 
| ![with shaded subsections](https://user-images.githubusercontent.com/6887128/36868423-46266624-1d66-11e8-89a5-3d3b362be1fa.png) | ![without shaded subsections](https://user-images.githubusercontent.com/6887128/36868424-4637de5e-1d66-11e8-9169-52694d20c8b2.png) | 

## Primary button 
Background color for hover and focus states
- Active state will need to be looked at too so that there is a unique active and hover/focused state
- The only accessible options for white text on red are Dark Pacific and Pacific So we will not be able to accommodate a 3 separate Pacifics for default, hover/focus, and active. We can bring in a Navy, however.  

| Current (Pacific 80 - #4497dc) | Recommendation (Dark Pacific - #0050B4) | 
| --- | --- | 
| ![with shaded subsections](https://user-images.githubusercontent.com/6887128/36868594-ce1d3a76-1d66-11e8-9f79-9108ad2b6ad9.png) | ![without shaded subsections](https://user-images.githubusercontent.com/6887128/36868579-c0d10ca8-1d66-11e8-9b35-b1ddc8ecc1a5.png) | 

## Destructive button 
Background color for hover and focus states
- Active state will need to be looked at too so that there is a unique active and hover/focused state.
- The only accessible options for white text on red are Dark Red and Red. So we will not be able to accommodate a 3 separate reds for default, hover/focus, and active. 

| Current (Red 80 - #dd735d) | Recommendation (Dark Red - #B63014) | 
| --- | --- | 
| ![with shaded subsections](https://user-images.githubusercontent.com/6887128/36868783-6b7995b2-1d67-11e8-9a27-7ea8ec3b3740.png) | ![without shaded subsections](https://user-images.githubusercontent.com/6887128/36868784-6b88de5a-1d67-11e8-8e5e-94d38794e698.png) |

**Comment by Scotchester** [comment]
18F:

> Note: Logos, **disabled form fields**, and **disabled buttons** are exempt from [color contrast guidelines] and don't need to be tested for contrast.

(Emphasis added.)


I agree that we should try to achieve them if possible, but if it's impossible to do so while having them visually still look disabled… :man_shrugging:

**Comment by nataliafitzgerald** [comment]
Yeah, that makes sense @Scotchester. I'm ok with making an exception for the disabled states so that they still maintain their visual meaning (i.e. look disabled).

**Comment by nataliafitzgerald** [comment]
@Scotchester 
Here's a mock-up of what we've discussed: 
<img width="403" alt="screen shot 2018-10-23 at 2 57 24 pm" src="https://user-images.githubusercontent.com/6887128/47387599-d685a780-d6dd-11e8-86ac-b28cec7da326.png">
